notebooks/chat-read-retrieve-read.ipynb

```diff
@@ -20,10 +20,7 @@
     "AZURE_OPENAI_SERVICE = os.environ.get(\"AZURE_OPENAI_SERVICE\") or \"myopenai\"\n",
     "AZURE_OPENAI_GPT_DEPLOYMENT = os.environ.get(\"AZURE_OPENAI_GPT_DEPLOYMENT\") or \"davinci\"\n",
     "AZURE_OPENAI_CHATGPT_DEPLOYMENT = os.environ.get(\"AZURE_OPENAI_CHATGPT_DEPLOYMENT\") or \"chat\"\n",
-    "\n",
-    "KB_FIELDS_CONTENT = os.environ.get(\"KB_FIELDS_CONTENT\") or \"content\"\n",
```

Comment:
Aren't KB_FIELDS_CONTENT and KB_FIELDS_SOURCEPAGE used in line 125?

Reply:
Thanks that was a miss . Updated.